client/verta/tests/time_strategies.py

```diff
@@ -1,11 +1,18 @@
 # -*- coding: utf-8 -*-
 
-import hypothesis.strategies as st
 from datetime import timedelta
 
-from verta._internal_utils.time_utils import timedelta_millis
+import hypothesis.strategies as st
+from verta._internal_utils.time_utils import duration_millis
+import warnings
+
+
+def duration_millis_ignore_warn(delta):
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")
+        return duration_millis(delta)
```

Comment:
It feels a bit odd to me to suppress warnings at the `hypothesis` strategy level, like a mixture of separate concerns? If it's to avoid polluting test output, what would you think about [using `pytest`](https://docs.pytest.org/en/stable/warnings.html#pytest-mark-filterwarnings) to filter out the timezone conversion warning, instead?

Reply:
This is in here basically as a convenience function for the strategy, because the hypothesis generator of time deltas _will_ generate time deltas with microseconds, and we use that generator as the basis for generating valid utc millisecond values. Doing it here is preferable in my opinion because in several of the tests I instead want to expect that the warnings will be produced.

Reply:
Ahhh, I see. Yes, that makes sense to me now as well.